src/platform/forms-system/src/js/web-component-patterns/fullNamePattern.js

```diff
@@ -212,7 +212,7 @@ const fullNameSchema = commonDefinitions.fullName;
  * @returns `commonDefinitions.fullName` minus `middle`
  */
 const firstNameLastNameDef = { ...commonDefinitions.fullName };
-delete firstNameLastNameDef.middle;
+delete firstNameLastNameDef.properties.middle;
```

Comment:
I think this might be deleting it from the source object (`commonDefinitions.fullName`), since { ...obj } is a shallow copy. maybe instead could just make a full object of the properties used, and shallow copy each property.

Reply:
Refactored with lodash cloneDeep.  Thanks for catching this!